galette/lib/Galette/IO/PdfMembersCardsAdaptative.php

```diff
@@ -195,16 +195,24 @@ public function drawCards(array $members): void
 
             // Color=#8C8C8C: Shadow of the year
             $this->SetTextColor(140);
-            $this->SetFontSize($this->year_font_size);
 
             //Write shadow of the year, center of the card
             $an_cot = $this->an_cot;
             if ($an_cot === 'DEADLINE') {
                 //get current member deadline
                 $an_cot = $member->due_date;
             }
-
-            $xan_cot = $x0 + $this->wi / 2 - $this->GetStringWidth(
+            if (is_null($an_cot)) {
```

Comment:
This condition does not work. I do not know exactly which problem you tried to fix here; but there is something wrong.

Reply:
I've been able to reproduce and fix.